apps/desktop/tests/utils/secretLikeContent.test.ts

```
@@ -0,0 +1,66 @@
+import { describe, expect, it } from 'vitest';
+
+import {
+    REDACTED_SECRET_LIKE_CONTENT,
+    containsSecretLikeContent,
+    redactAllStringValues,
+    redactSecretLikeContent,
+    redactSecretLikeStringValues,
+} from '@/utils/secretLikeContent';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)